1208-projects/project_wizard.feature

```diff
@@ -0,0 +1,190 @@
+Feature: User edits Project
+    In order to document a project 
+    As a logged in User
+    I want to edit existing and create new projects
+
+    @xxx
+
+    Scenario Outline: User sees the Project Wizard
+        Given User opens My Projects
+        When they click Create New Project Button
+        Then the User is the Project Owner
+        And they see tabs:
+            | Description  |
+            | Participants |
+            | Financing    |
+        And they see the Description tab
+        And they see fields:
+            | Title                    |
+            | Summary                  |
+            | Coordinating Institution |
+            | Start Date               |
+            | End Date                 |
+            | Internal Project Code    |
+        And they see a Support Button
+        And they see a Button "<Persist>" desided by Project's "<Status>"
+        And they see a Next Button
+        Examples:
+            | Status    | Persist |
+            | Published | Publish |
+            | Draft     | Save    |
+
+    @xxx
+    Scenario: Curator opens a Project in the Project Wizard
+        # Same end result as "User Edits a Project in the Project Wizard"
+        # but this one enables a Curator to edit projects where he's not an
+        # participant
+        Given User opens Landing Page for Project
+        And User is Curator on Project's Project Owner's Institution
+        And the project lacks an Approval of type "REK"
+        # A "REK" Approved project is a Health Project. 
+        # See health_related_projects.feature for details.
+        When they click the Edit button
+        Then User sees the Project Wizard
+        And it contains data about the Project
+
+    @xxx
+    Scenario Outline: User opens Participants tab for Project
+        Given User sees the Project Wizard
+        And User has the "<Role>" role in the project
+        When they click the Participants tab
+        Then they see lists of:
+            | Project Managers     |
+            | Project Participants |
+        And they see Buttons:
+            | Add Project Manager     |
+            | Add Project Participant |
+        And they see a Previous Button
+        And they see a Support Button
+        And they see a Button "<Persist>" desided by Project's "<Status>"
+        And they see a Next Button
+        Examples:
+            | Role                  |
+            | Curator               |
+            | Project Owner         |
+            | Project Manager       |
+        Examples:
```

Comment:
Usikker på om det er lov med 2stk `Examples` i samme scenario..? @Eiriknil

Reply:
`desided` skal være `decided`

Reply:
Er usikker, men kan prøve å finne ut av det

Reply:
Nei, de to tabellene må slås sammen